lego/apps/events/models.py

```diff
@@ -755,40 +793,40 @@ class Pool(BasisModel):
     class Meta:
         ordering = ["id"]
 
-    def delete(self, *args, **kwargs):
+    def delete(self, *args: Any, **kwargs: Any) -> tuple[int, dict[str, int]]:
         if not self.registrations.exists():
-            super().delete(*args, **kwargs)
+            return super().delete(*args, **kwargs)
```

Comment:
Assuming this won't affect any logic? `super().delete()` returns a tuple further up, making it wrong to say this returns `None`, thus making mypy complain that there's no return statement.